aito/schema.py

```diff
@@ -866,8 +930,7 @@ def __init__(self):
         super().__init__('String')
 
     def to_python_type(self):
-        return str
-
+       return str
```

Comment:
Incorrect indentation :open_mouth: It starts with one broken window...

```suggestion
        return str
```